app/controllers/admin/users_controller.rb

```diff
@@ -47,5 +48,11 @@ def update
     def sign_params
       params.require(:user).permit(:name, :email, :password, :password_confirmation, :phone)
     end
+    def require_same_user
+      if current_user != User.find(params[:id]) and !current_user.admin?
```

Comment:
może zamiast tego zrobić: `return unless current_user != User.find(params[:id]) and !current_user.admin?`